src/tools/rbd_mirror/group_replayer/Replayer.cc

```diff
@@ -536,6 +572,15 @@ void Replayer<I>::validate_image_snaps_sync_complete(
 
     // In case if the image replayer is not yet updated with the limits, do it here
     if (!image_snap_complete) {
+      if (m_snapshot_start.is_zero()) {
+	// The mirror group snapshot's start time being zero and the member
+	// image snapshots being incomplete indicate that the group replayer
+	// was restarted interrupting the sync of the mirror group snap. So
+	// reset the group snap's start time that is used to calculate
+	// the total time taken to complete the syncing of the group snap
+	// as best as we can.
+	m_snapshot_start = ceph_clock_now();
```

Comment:
@pkalever does this look like the right place to reset the start time after GroupReplayer restart?